cluster-operator/src/main/java/io/strimzi/operator/cluster/PlatformFeaturesAvailability.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2017-2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster;
+
+import io.strimzi.operator.cluster.operator.KubernetesVersion;
+
+
+public class PlatformFeaturesAvailability {
+
+    private boolean isOpenshift;
+    private final KubernetesVersion kubernetesVersion;
+
+    public PlatformFeaturesAvailability(boolean isOpenshift, KubernetesVersion kubernetesVersion) {
+        this.isOpenshift = isOpenshift;
+        this.kubernetesVersion = kubernetesVersion;
+    }
+
+    public boolean isOpenshift() {
+        return this.isOpenshift;
+    }
+
+    public int getMinorVersion() {
```

Comment:
You don't need these methods for getting the major and minor version: Just a getter for the `kubernetesVersion`.